wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -144,18 +147,33 @@ getAccountsIncludeUnready
 getAccountsIncludeUnready includeUnready mCAddr = do
     whenJust mCAddr $ \cAddr -> getWalletMetaIncludeUnready includeUnready cAddr `whenNothingM_` noWallet cAddr
     accIds <- maybe getAccountIds getWalletAccountIds mCAddr
+    logInfo $ sformat ("getAccountsIncludeUnready: Computing balance for " % shown) accIds
     let groupedAccIds = fmap reverse $ HM.fromListWith mappend $
                         accIds <&> \acc -> (aiWId acc, [acc])
+    logInfo $ sformat ("getAccountsIncludeUnready: Grouped AccountId(s) " % shown) groupedAccIds
     balAndUtxo <- WS.getWalletBalancesAndUtxo
-    concatForM (HM.toList groupedAccIds) $ \(wid, walAccIds) ->
-         fixCachedAccModifierFor wid $ \accMod ->
-             mapM (getAccountMod balAndUtxo accMod) walAccIds
+    logInfo $ sformat ("getAccountsIncludeUnready: " % stext) (renderUtxo    . snd $ balAndUtxo)
+    logInfo $ sformat ("getAccountsIncludeUnready: " % stext) (renderBalance . fst $ balAndUtxo)
```

Comment:
Same prefix in these two logs

Reply:
That is intentional (unless I have done something stupid): the idea there was to track in the prefix the function this logging was coming from, just for making the reconstruction process a bit easier :)

Reply:
Yes, I mean, you omitted sub-function tag (e.g. `getAccountsIncludeUnready: balance`), is it for purpose?

Reply:
Fair point, but if you look at the implementations of `renderUtxo` and `renderBalance` they both print what is being logged (i.e. either the balance or the utxo). Am I missing something? 😀

Reply:
Right :man_facepalming: